src/Symfony/Component/BrowserKit/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.1.0
```

Comment:
5.3.0

Reply:
5.3? Is 5.2 in feature freeze? Think the change in 5.2 on the request params of would make it great that this would be in the same release. As this should make it easier to refractor the api tests, which will fail with 5.2 new interpration of the browser kit parameters.

Reply:
yes. feature freeze was at the end of September